compiler/AST/build.cpp

```diff
@@ -378,11 +396,63 @@ static BlockStmt* buildUseList(BaseAST* module, BlockStmt* list) {
   }
 }
 
+//
+// Given a string literal argument from a 'use' statement, process
+// that argument.  We assume it's either a "-llib" flag,
+// or a filename like "foo.h", "foo.c", "foo.o", etc.  
+//
+// - For the former, pass to addLibInfo(), the same function that
+//   handles command line -l flags.
+//
+// - Otherwise, assume it's the latter and pass it to our input file
+//   handler (which itself handles cases it doesn't recognize).
+//
+static void processUseOfString(const char* str) {
+  // If the string starts with -l, treat it as a library specifier
+  //
+  if (strncmp(str, "-l", 2) == 0) {
+    addLibInfo(str);
+  } else {
+    //
+    // otherwise, treat it as an input file
```

Comment:
I think you can remove this comment, the code is pretty self explanatory now (if you like the comment, 'input file' should be changed to 'source file')